CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ target_link_options(
         -sEXPORT_ES6
         -sMODULARIZE
         -sWASM_BIGINT
+        --emit-tsd ${CLP_FFI_JS_BIN_NAME}.d.ts
```

Comment:
The `.d.ts` file basename should match the basename of the genrated js file so that any import of such
```
import clpFfijsModuleInit, {ClpIrStreamReader} from "path/to/ClpFfiJs";
```
also imports the tsd file.
Note how the filename extension `.js` can be omitted in the import.